src/hb-ot-var-common.hh

```diff
@@ -1938,6 +1938,35 @@ struct item_variations_t
 
   /* main algorithm ported from fonttools VarStore_optimize() method, optimize
    * varstore by default */
+
+  struct combined_gain_idx_tuple_t
+  {
+    int gain;
+    unsigned idx_i;
+    unsigned idx_j;
```

Comment:
Is there a better more descriptive name for these indices than i and j?

Reply:
they represent the indices of 2 encodings in an array that we want to combine. Would idx_1, idx_2 be better?

Reply:
Got it, yeah idx_1/2 seems fine.